cassandra/cassandra-docker.sh

```diff
@@ -130,6 +130,51 @@ if [ -n "$HELP" ]; then
   exit 0
 fi
 
+if [ -z "${MAX_HEAP_SIZE}" ] && [ -z "$MEMORY_LIMIT" ]; then
+  echo "The MAX_HEAP_SIZE and MEMORY_LIMIT envars are not set. Defaulting the MAX_HEAP_SIZE to 512M"
+  export MAX_HEAP_SIZE=512M
```

Comment:
Just asking the question because I don't know the answer, is it safe to assume we will always have 512M?  This block essentially says, "you have not told me your desired heap, and i have not been informed of the limit". Alternative would be to also check the cgroup limit?